lib/utils.js

```diff
@@ -34,6 +34,15 @@ function init(ctx) {
     }
   };
 
+  utils.toFixedMin = function toFixedMin(value,digits) {
+    if (value == null || isNaN(value)) {
```

Comment:
`== null` checks aren't very reliable, this would probably be more robust as `if (!value || isNaN(value)) {`?